node-fs-writefile/cp.js

```diff
@@ -0,0 +1,10 @@
+import { writeFile } from 'node:fs/promises';
+import { readFile } from 'node:fs/promises';
+
+try {
+  const filePath = new URL(process.argv[2], import.meta.url);
```

Comment:
This works, but it's not really necessary for this exercise. This code produce an absolute file path from the relative file path in `process.argv[2]`. You can just pass the relative file path directly to `readFile`.